packages/js/data/src/crud/utils.ts

```diff
@@ -148,6 +149,52 @@ export const getUrlParameters = (
 	return params;
 };
 
+/**
+ * Check to see if an argument is a valid type of ID query.
+ *
+ * @param  arg       Unknow argument to check.
+ * @param  namespace The namespace string
+ * @return boolean
+ */
+export const isValidIdQuery = ( arg: unknown, namespace: string ) => {
```

Comment:
Very nitpick: Does make sense to expect `arg` to be a string, number, or object?

```
arg: string | number | object
```

Reply:
I think because this gets run on all actions, we really aren't sure what it is and it could be any data type.  E.g., `myCustomAction( true )`

We have some control over actions inside the CRUD data store, but the hope is that this will be useful to CRUD data stores that extend the functionality.